src/electron/ipc/ipc-event-action.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export class IpcEventAction {
+    private listeners: (() => Promise<void>)[] = [];
+
+    public async invokeAsync(): Promise<void> {
```

Comment:
This may need to be an arrow function to get the correct `this`. Unit tests will make that clear

Reply:
I think it is OK, I've compared it to the methods I see in [Action](https://github.com/microsoft/accessibility-insights-web/blob/master/src/common/flux/action.ts), can you let me know if I'm missing something in particular?

Reply:
If unit tests are happy and it enables the E2E tests, I'm good :)